### DISCUSSION TITLE
Asking the main developer about JavaScript

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#590] [Q&A] [answered]
Do you know JavaScript programming (at least basic lessons)? I am asking this to you, because you said in one of my issues “I think it's an impossible range (requiring JavaScript) in this project.”.

**Comment by black7375** [comment, marked as answer]
Yes. see this.
https://github.com/black7375/font-range

**Reply by Aleksandra0056** [reply]
Thank you for quick response. My another question is that: Why did you avoid JavaScript in this project (Firefox UI Fix)? JS is there only 1.7%, but why?

**Reply by black7375** [reply]
To use JS, installation is difficult, and there is a security problem by [disabling the sandbox](https://github.com/MrOtherGuy/fx-autoconfig/blob/master/program/defaults/pref/config-prefs.js).

https://github.com/black7375/Firefox-UI-Fix/blob/master/docs/Preference.md#auto-config

**Comment by Aleksandra0056** [comment]
I'm closing this. I want to say again thank you for the response.
